src/amo/components/AddAddonToCollection/styles.scss

```diff
@@ -4,4 +4,5 @@
 
 .AddAddonToCollection-added {
   margin-bottom: 10px;
+  word-break: break-word;
```

Comment:
This is deprecated but still supported. I suggest switching it to [`word-break: break-all`](https://developer.mozilla.org/en-US/docs/Web/CSS/word-break) which will work better in certain situations.